swagger/sales/coupons.json

```diff
@@ -36,55 +52,37 @@
             ]
           }
         ],
-        "parameters": [
-          {
-            "name": "Authorization",
-            "in": "header",
-            "type": "string",
-            "required": true,
-            "description": "The format of the API authorization header should be: 'Authorization': \"Bearer API_TOKEN\"",
-            "x-example": "Bearer abc...123"
-          }
-        ],
-        "x-amazon-apigateway-auth": {
-          "type": "NONE"
-        },
-        "x-auth-type": "Application & Application User",
         "responses": {
           "200": {
             "description": "Success",
             "examples": {
               "application/json": {
                 "success": true,
                 "data": {
-                  "success": true,
-                  "data": {
-                    "success": "true / false",
-                    "data": [
-                      {
-                        "uid": "wo9jvxhpb9fqmrqp",
-                        "business_uid": "k72pvtwaui4pa05v",
-                        "coupon_name": "General 20% OFF coupon",
-                        "discount_type": "percent",
-                        "discount_value": 20.0,
-                        "coupon_code": "SNT6CDBWKY",
-                        "status": "active",
-                        "valid_for_services": [
-                          "service_uid1"
-                        ],
-                        "valid_for_staff": [
-                          "staff_uid1"
-                        ],
-                        "start_date": "2024-04-21T11:35:52.000+03:00",
-                        "expiration_date": "2025-04-21T11:35:52.000+03:00",
-                        "max_redemptions": 25,
-                        "max_redemptions_per_client": 1,
-                        "total_redemptions": 0,
-                        "created_at": "2024-04-21T11:35:52.000+03:00",
-                        "updated_at": "2024-04-21T11:35:52.000+03:00"
-                      }
-                    ]
-                  }
+                  "coupons": [
+                    {
+                      "uid": "wo9jvxhpb9fqmrqp",
```

Comment:
you are not supposed to have values here. you are supposed to have a reference to the entity schema:

"PermissionsList": {
                "type": "object",
                "properties": {
                    "premissions": {
                        "description": "List of Permissions",
                        "type": "array",
                        "items": {
                            "$ref": "https://vcita.github.io/developers-hub/entities/access_control/permission.json"
                        }

                    }
                }
            },
---------
"responses": {
                    "200": {
                        "description": "",
                        "content": {
                            "application/json": {
                                "schema": {
                                    "allOf": [
                                        {
                                            "$ref": "https://vcita.github.io/developers-hub/entities/response.json"
                                        },
                                        {
                                            "properties": {
                                                "data": {
                                                    "type": "object",
                                                    "$ref": "#/components/schemas/PermissionsList"
                                                }
                                            }
                                        }
                                    ]
                                }
                            }
                        }
                    }
                },